### PR TITLE
create new class

### PR DESCRIPTION
**Pull request by tramphan323**
:D

### CODE REVIEW

File: src/main/java/tram/stringreverse/StringReverse.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package tram.stringreverse;
+
+/**
+ * Created by Tram on 7/8/2015.
+ */
+public class StringReverse {
```
Comment (trungph) [inline src/main/java/tram/stringreverse/StringReverse.java:6]: Create new method:
public String reverse(String text) {
    return new StringBuilder(text).reverse().toString();
}

**Comment by trungph** [comment]
OK good, you need to add a test class according to the assignment instruction

File: src/main/java/tram/stringreverse/StringReverse.java (hunk 1 above)
Comment (tramphan323) [inline src/main/java/tram/stringreverse/StringReverse.java:6]: added

**Comment by tramphan323** [comment]
test class added

**Comment by trungph** [comment]
Pull request is merged.

**Pull request closed without merging**